apps/balancer-tools/src/app/apr/api/(utils)/fetchDataForPoolIdDateRange.ts

```diff
@@ -23,8 +22,11 @@ export async function fetchDataForPoolIdDateRange(
 
     while (attempts < MAX_RETRIES) {
       try {
-        const currentRound = Round.getRoundByDate(dayDate);
-        const data = await calculatePoolStats({ round: currentRound, poolId });
+        const data = await calculatePoolStats({
+          startAt: new Date(dayDate.getTime() - 24 * 60 * 60 * 1000),
+          endAt: dayDate,
```

Comment:
🤔 why always calculate starting at a day before? Also -- shouldn't we change this signature to only get the unix timestamp altogether? 😅 it feels we're transforming this back and forth all the time